jobs/service-fabrik-bosh-manager/templates/config/settings.yml.erb

```diff
@@ -0,0 +1,288 @@
+<%
+  def ostruct_to_hash(os)
+    os.marshal_dump.map do |key, value|
+      { key => value.is_a?(OpenStruct) ? ostruct_to_hash(value) : value }
+    end.reduce(&:merge)
+  end
+
+  broker_name = link("broker").p('name')
+  log_path = "/var/vcap/sys/log/service-fabrik-bosh-manager/service-fabrik-bosh-manager.log"
+  networks = ostruct_to_hash(spec.networks)
+  default_ip = networks.values.find { |net| net.has_key?(:default) }[:ip]
+  external = link("broker").p('external')
+  internal = link("broker").p('internal')
+%>---
+production:
+  ####################
+  # GENERAL SETTINGS #
+  ####################
+  broker_name: '<%= broker_name %>'
+  deployment_name: '<%= spec.deployment %>'
+  release_name: '<%= spec.release.name %>'
+  release_version: '<%= spec.release.version %>'
+  username: '<%= link("broker").p('username') %>'
+  password: '<%= link("broker").p('password') %>'
+  broker_ip: <%= spec.ip %>
+  enable_service_fabrik_v2: <%= link("broker").p('enable_service_fabrik_v2') %>
+  skip_ssl_validation: <%= link("broker").p('skip_ssl_validation') %>
+  session_store:
+    path: <%= "/var/vcap/store/#{broker_name}/session" %>
+  log_path: <%= log_path %>
+  log_level: <%= link("broker").p('log_level') %>
+  sys_log_level: <%= link("broker").p('sys_log_level') %>
+  enable_bosh_rate_limit: <%= link("broker").p('enable_bosh_rate_limit') %>
+  enable_circuit_breaker: <%= link("broker").p('enable_circuit_breaker') %>
+  enable_swarm_manager: <%= link("broker").p('enable_swarm_manager') %>
+  feature:
+    ServiceInstanceAutoUpdate: <%= link("broker").p('feature.ServiceInstanceAutoUpdate') %>
+    EnableSecurityGroupsOps: <%= link("broker").p('feature.EnableSecurityGroupsOps') %>
+  http_timeout: <%= link("broker").p('http_timeout') %>
+  deployment_action_timeout: <%= link("broker").p('deployment_action_timeout') %>
+  multi_az_enabled: <%= link("broker").p('multi_az_enabled') %>
+  ha_enabled: <%= link("broker").p('ha_enabled') %>
+  broker_drain_message: <%= link("broker").p('broker_drain_message') %>
+  ##############################
+  # EXTERNAL ENDPOINT SETTINGS #
+  ##############################
+  external:
+    log_event: <%= link("broker").p('external.log_event') %>
+    event_type: SF.API_EVENT
+    trust_proxy: <%= external['trust_proxy'] %>
+    port: <%= external['port'] %>
+    <% link("broker").if_p('external.ssl') do |ssl| %>
+    protocol: 'https'
```

Comment:
Can you put like this?
   protocol: 'https'
   <% link("broker").if_p('external.ssl') do |ssl| %>
    ssl: <%= JSON.dump(ssl) %>
    <% end %> 

else part not required.

Reply:
done.